src/components/Select/Select.stories.tsx

```diff
@@ -0,0 +1,223 @@
+import { BADGE } from '@geometricpanda/storybook-addon-badges';
+import type { StoryObj, Meta } from '@storybook/react';
+import { within } from '@storybook/testing-library';
+import clsx from 'clsx';
+import React from 'react';
+import type { OptionsAlignType, VariantType } from './Select';
+import { Select } from './Select';
+import styles from './Select.stories.module.css';
+import Icon from '../Icon';
+
+export default {
+  title: 'Molecules/Forms/Select',
+  component: Select,
+  parameters: {
+    badges: [BADGE.BETA],
+    layout: 'centered',
+  },
+  decorators: [
+    (Story) => (
+      <div
+        style={{
+          margin: '1rem', // Provides spacing to see activity around the select field
```

Comment:
If layout is `centered` does this still make a difference?

Reply:
Yah, it doesn't seem necessary anymore. I will remove this decorator to simplify things